tests/fish_title.test.fish

```diff
@@ -29,5 +23,12 @@ end
     fish_title
 ) = "/tmp/current/directory - fish"
 
+@test "fish_title: contains shorten (1) current path" (
+    set --universal pure_title_dir_length 1
+    fish_title
+) = "/t/c/directory - fish"
 
-teardown
+@test "fish_title: contains shorten (2) current path" (
+    set --universal pure_title_dir_length 2
+    fish_title
+) = "/tm/cu/directory - fish"
```

Comment:
Thanks for the tests! 
We can move those to `./tests/_pure_parse_directory.test.fish`